native/dlasq5.go

```diff
@@ -0,0 +1,122 @@
+// Copyright ©2015 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package native
+
+import "math"
+
+// Dlasq5 computes one dqds transform in ping-pong form.
+// i0 and n0 are zero-indexed.
+func (impl Implementation) Dlasq5(i0, n0 int, z []float64, pp int, tau, sigma, dmin, dmin1, dmin2, dn, dnm1, dnm2 float64) (i0Out, n0Out, ppOut int, tauOut, sigmaOut, dminOut, dmin1Out, dmin2Out, dnOut, dnm1Out, dnm2Out float64) {
```

Comment:
It's not that simple. You'll notice that the first return happens, and that in the LAPACK function none of the values are initialized. Well... since Fortran passes by reference their value is preserved through the call. It seems like they are actually outputs here (the tests don't fail if I add it), but I ran into these bugs in Dlasq4. I added a TODO.